api/src/main/java/org/pmiops/workbench/interceptors/InterceptorUtils.java

```diff
@@ -16,7 +23,14 @@ public static Method getControllerMethod(HandlerMethod handlerMethod) {
     Pattern apiControllerPattern = Pattern.compile("(.*\\.[^.]+)Api(Controller)");
     Method apiControllerMethod = handlerMethod.getMethod();
     String apiControllerName = apiControllerMethod.getDeclaringClass().getName();
-    String controllerName = apiControllerPattern.matcher(apiControllerName).replaceAll("$1$2");
+
+    String controllerName;
```

Comment:
I assumed this was some temporary change for your oneoff branch - but I guess this actually is due to moving controllers out of the API directory. This deserves a comment